resources/config.edn.sample

```diff
@@ -0,0 +1,7 @@
+{:datahike {:store {:backend :file
```

Comment:
We will have multiple stores in the future, so this will be a vector. But for now, this is fine.